app/oauth.py

```diff
@@ -12,6 +12,8 @@
 
 BACKEND_URL = os.getenv("BACKEND_URL")
```

Comment:
Avoid global variables. Use this value where you need it and check that it is not empty before using it